### DISCUSSION TITLE
Is there a way to train model on GPU without transformers?

### DISCUSSION THREAD

**Discussion by erotavlas** [explosion/spaCy#7724] [Help: Other Questions] [answered]
In the starter config from the main website, it seems there is no option to initialize the config for GPU (without transformers), there is only two options CPU, and GPU(transformers).  If we want to train just regular model is there a sample config available showing how to do it?  I tried disabling the transformers pipeline but it failed with an error during training.

**Comment by polm** [comment, marked as answer]
You can pass `--gpu-id 0` (or your actual gpu id) to the `spacy train` command to train on GPU.

I think the reason that this isn't included in the start configs is that if you have a GPU, the boost in accuracy from Transformers is generally worth using.
